src/include/sof/audio/coefficients/volume/windows_fade.h

```diff
@@ -0,0 +1,56 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ */
+
+#include <sof/audio/format.h>
+#include <stdint.h>
+
+#define WINDOWS_FADE_LUT_SIZE 128
+#define WINDOWS_FADE_LUT_STEP \
+	Q_CONVERT_FLOAT(1.0 / (WINDOWS_FADE_LUT_SIZE - 1), 30)  /* Q2.30 */
+
+/* Calculate power y = x^1.75, input and output are in Q2.30 format */
+
+static const int16_t windows_fade_lookup[WINDOWS_FADE_LUT_SIZE] = {
```

Comment:
The lookup table is 256 bytes, maybe this should be a kconfig option?

Reply:
yes, needs to be a kconfig.

Reply:
Done